enzy_htp/structure/structure_io/__init__.py

```diff
@@ -11,4 +11,4 @@
 #         module_name = f.removesuffix(".py")
 #         __all__.append(module_name)
 #         import_module("."+module_name, package="core.clusters")
-# del f # pylint: disable=undefined-loop-variable
```

Comment:
Even though Google style guide didn't mention it explictly. The pylintrc it provides always tells me to add a newline at the bottom of a file. I think it is good.

Reply:
Good to know!